cli/main.rs

```diff
@@ -80,4 +86,5 @@ fn repl(mut jstime: jstime::JSTime) {
             }
         }
     }
+    rl.save_history(&history_path).unwrap();
```

Comment:
this would crash when history fails to be written, is that desirable?

Reply:
No, no it isn't. Shouold we do the same variable assignment as above to swallow the error?

Reply:
should be sufficient